client/src/components/EditBox.js

```diff
@@ -17,21 +17,19 @@ export default function EditBox() {
   console.log(skills.length);
   const [deleted, setDeleted] = useState(skills);
   const [learningObjective, setLearningObjective] = useState(skills);
-  const [learningObj, setLearningObj] = useState([])
+  const [learningObj, setLearningObj] = useState([]);
```

Comment:
`[learningObj, setLearningObj]` line 20, looks very similar to `[learningObjective, setLearningObjective]` line 19. Are they both used for different things? Can we use more meaningful names to make it easier to understand?